CHANGELOG.md

```diff
@@ -2,11 +2,55 @@
 
 ## Table of Contents
 
+- [v0.3.0](#v030)
 - [v0.2.0](#v020)
 - [v0.1.0](#v010)
 - [v0.1.0-rc2](#v010-rc2)
 - [v0.1.0-rc1](#v010-rc1)
 
+## v0.3.0
+
+API Version: v1alpha1
+
+### API changes
+
+#### Gateway
```

Comment:
@robscott should we indicate that route selectors are now a pointer?

xref: https://github.com/kubernetes-sigs/gateway-api/commit/dc82c09b08b816fb786004eb6af58ac993c2bb34